src/components/Audio/Audio.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faVolumeUp, faSpinner } from "@fortawesome/free-solid-svg-icons";
+
+import "./Audio.scss";
+
+interface LocalProps {
+    url: string;
+}
+type Props = LocalProps;
+
+interface LocalState {
+    play: boolean;
+}
+
+class AudioComponent extends React.Component<Props, LocalState> {
+    private audioElem: HTMLAudioElement;
+    constructor(props) {
+        super(props);
+        this.state = {
+            play: false
```

Comment:
shouldn't this be `playing` or `isPlaying` rather than just `play`?

Reply:
Done